packages/slice-machine/components/Simulator/components/SetupModal/CodeBlock.tsx

```diff
@@ -32,10 +40,15 @@ const CodeBlockWithCopy: React.FC<{
         {...(lang ? { lang, codeClass: "is-json" } : {})}
         sx={{
           width: "100%",
-          height: "100%",
+          minHeight: "100%",
           bg: "codeBlockBackground",
           borderRadius: "6px",
           flexDirection: "column",
+          ...(fullHeightCode
```

Comment:
unsure of what that would change, should have been better to pass an `sx` object ?